backend/python/app/middlewares/auth.py

```diff
@@ -149,16 +149,31 @@ def wrapper(root, info, *args, **kwargs):
     return require_authorization
 
 
-def require_authorization_as_story_translator():
+def require_authorization_as_story_user_by_role(as_translator):
     def require_authorization(api_func):
         @wraps(api_func)
         def wrapper(root, info, *args, **kwargs):
             access_token = get_access_token(info.context)
-            authorized = auth_service.is_translator(
-                access_token, kwargs["story_translation_contents"][0]["id"]
+            # TODO: find generalized way of getting story_translation_content_id.
+            # Currently custom to UpdateStoryTranslationContentStatus and UpdateStoryTranslationContents
```

Comment:
just wanted to add that it might be problematic if we have a case where `story_translation_contents` isn't in kwargs either, e.g. if only the story_translation's `id` is passed in

not sure which mutations fit that use case but it's just something to note!